docs/courses/coursemanagement.md

```diff
@@ -254,6 +254,18 @@ You can now
 ![Book Mapping](/img/bookmapping.png)
 
 ## Publishing
+
+As you update your assignment, a **Publish** button will show to the left of your username in the top menu bar.  
```

Comment:
"will show on the top menu bar, to the left of your username"

Please include the general area of the screen BEFORE the more detailed exact location -- easier to follow

Reply:
text updated, do not understand 2nd comment. Generaly throughout the docs, the images are shown under the description so if you mean to move the image above the words will not be consistent approach

Reply:
No -- the second is a general rule for describing the location of UI elements -- start with the biggest grain size of information and work your way to smaller grain size (i.e. on the left in the file tree -- above the file list, in the top tool bar -- to the left of your username, etc). You are trying to get them in the correct region of the screen before giving them the next level of specificity.